types/node/index.d.ts

```diff
@@ -5915,6 +5915,13 @@ declare module "tls" {
          * May be used to distinguish TLS sockets from regular ones.
          */
         encrypted: boolean;
+
+        /**
+         * String containing the selected ALPN protocol.
+         * When ALPN has no selected protocol, tlsSocket.alpnProtocol equals false.
+         */
+        alpnProtocol: string;
```

Comment:
I don't find the `alpnProtocol` here.Are you sure this needs it?

Reply:
@isLishude it's used in the http2 documents for when checking whether to use push or not.

https://nodejs.org/api/http2.html#http2_alpn_negotiation

Reply:
This should be `alpnProtocol?: string` then?

Reply:
Fixed